lib/console.js

```diff
@@ -86,7 +86,12 @@ function createWriteErrorHandler(stream) {
   };
 }
 
-function write(ignoreErrors, stream, string, errorhandler) {
+function write(ignoreErrors, stream, string, errorhandler, groupIndent) {
+  if (groupIndent.length !== 0) {
```

Comment:
You can also check for `\n` first by adding a indexOf guard. Using indexOf for cases were there is no newline it is a lot faster and almost negligible in case there is one.
```
if (groupIndent.length !== 0 && string.indexOf("\n") !== -1) {
  // ...
```
(Yes, indexOf is still faster then includes)